src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -9747,9 +9747,9 @@ static ref S<T> F2<T>(S<T> x2)
             if (languageVersion == LanguageVersion.CSharp10)
             {
                 comp.VerifyEmitDiagnostics(
-                    // (8,28): error CS9063: UnscopedRefAttribute cannot be applied to this parameter because it is unscoped by default.
+                    // (8,28): warning CS9269: UnscopedRefAttribute is only valid in C# 11 or later or when targeting net7.0 or later.
```

Comment:
It feels like there is no need to stop reporting the original error as part of reporting this warning. I do not think it serves the user experience, for example, since if user updates TFM and recompiles, it's inconvenient to wait all the way thru a second build just to be presented with errors which could have been reported the first time. It's ok to just tell the user everything that we know to be wrong.

Reply:
Some tests changed as you suggest due to the change to decouple the error and warning reporting.

However, this particular test stays the same, but I think that's correct. As you say, "if user updates TFM and recompiles, it's inconvenient to wait all the way thru a second build just to be presented with errors which could have been reported the first time" - but here on TFM/LangVersion update, no new errors for this signature are reported (as can be seen in the `else` branch below).